tls/s2n_client_hello.c

```diff
@@ -292,7 +261,7 @@ static int s2n_populate_client_hello_extensions(struct s2n_client_hello *ch)
     return 0;
 }
 
-int s2n_client_hello_recv(struct s2n_connection *conn)
+int s2n_client_hello_recv_new(struct s2n_connection *conn)
```

Comment:
Is this used anywhere? Can't find the symbol anywhere else for some reason.

Reply:
Already refactored and removed this function in https://github.com/songhu/s2n/pull/5